warehouse/query-core/src/test/java/datawave/query/MaxExpansionRegexQueryTest.java

```diff
@@ -266,7 +266,9 @@ public void testMaxIvaratorResultsFailsQuery() throws Exception {
         runTest(query, expect);
         // verify that the ivarators ran and completed
         assertTrue(countComplete(dirs) >= 1);
-        
+
+        // clear list before new set is added
+        dirs.clear();
```

Comment:
How exactly is this supposed to help?  At line 273 we reassign the dirs variable so I am a little confused as to how this fixes the periodic failure.